### PR TITLE
IRCCloud Support ARM

### PR DESCRIPTION
**Pull request by lucyllewy**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew install --cask {{cask_file}}` worked successfully.
- [ ] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by diddledan** [comment]
The failure is because there are no marked-stable releases in GitHub on the irccloud repository - every release is currently marked as pre-release, however this is the expected version from upstream as is installed by automatic update if you have irccloud already installed.

File: Casks/irccloud.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  version "0.15.0"
-  sha256 "759f91f8ac67ed74119fde39a945c6a5cc4fb426e978910b7541a5c84b2df7ea"
+  version "0.16.0"
+  sha256 "e942432cc457ce275960f6e5f120cc7be914ee06dc45fc88f9566a9aae251d79"
 
-  url "https://github.com/irccloud/irccloud-desktop/releases/download/v#{version}/IRCCloud-#{version}.dmg"
+  url "https://github.com/irccloud/irccloud-desktop/releases/download/v#{version}/IRCCloud-#{version}-universal.dmg"
   appcast "https://github.com/irccloud/irccloud-desktop/releases.atom"
   name "IRCCloud Desktop"
+  desc "Wrapper of the  website in a native window"
```
Comment (vitorgalvao) [inline Casks/irccloud.rb:8]: What website? This seems like a generic Electron app description.

Comment (diddledan) [reply Casks/irccloud.rb:8]: It said `IRCCloud website` when I wrote it, but I think the style check must have stripped it.

Comment (vitorgalvao) [reply Casks/irccloud.rb:8]: That wouldn’t have been descriptive. A person looking at `desc` should have some sense of what the app is or does or what it lets you accomplish, not how it’s built.

File: Casks/irccloud.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+  sha256 "e942432cc457ce275960f6e5f120cc7be914ee06dc45fc88f9566a9aae251d79"
 
-  url "https://github.com/irccloud/irccloud-desktop/releases/download/v#{version}/IRCCloud-#{version}.dmg"
+  url "https://github.com/irccloud/irccloud-desktop/releases/download/v#{version}/IRCCloud-#{version}-universal.dmg"
   appcast "https://github.com/irccloud/irccloud-desktop/releases.atom"
   name "IRCCloud Desktop"
+  desc "IRC client"
   homepage "https://github.com/irccloud/irccloud-desktop"
 
   app "IRCCloud.app"
```
Comment (vitorgalvao) [inline Casks/irccloud.rb:11]: Are you able to provide a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md)? See [`homebrew-createzap`](https://github.com/nrlquaker/homebrew-createzap) for an external project which can help.

Comment (diddledan) [reply Casks/irccloud.rb:11]: Added in 0d6210a

File: Casks/irccloud.rb (hunk 1 above)
Comment (diddledan) [reply Casks/irccloud.rb]: Thanks for updating. Marking this thread resolved.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**